Apps/VisionOS/VisionOS/TutorialViews/SDKInstallationTutorialView.swift

```diff
@@ -0,0 +1,42 @@
+import MarkdownUI
+import SwiftUI
+
+struct SDKInstallationTutorialView: View {
+    static let title = ScreenTitleConfig("SDK Installation")
+    @ObservedObject var state = AppState.shared
+    @EnvironmentObject private var viewModel: ViewModel
+    var body: some View {
+        Markdown {
+            """
+            You can install CustomerIO Swift SDKs
```

Comment:
```suggestion
            You can install CustomerIO iOS SDKs
```

Similar comment as other PRs of making sure the name of our SDK is consistent with the app.